app/code/Magento/Sales/Model/Order/Address.php

```diff
@@ -508,207 +508,307 @@ public function getVatRequestSuccess()
     }
 
     /**
-     * {@inheritdoc}
+     * Set Parent ID
```

Comment:
Please, revert changes related to @inheritdoc. Replace {@inheritdoc} with @inheritdoc